install/installer/traf_add_sudoAccess

```diff
@@ -40,8 +40,8 @@ function backupRestore {
 
 source $TRAF_CONFIG
 
-echo "## Allow $BACKUP_USER id to run commands needed for backup and restore" > $BACKUP_USER
-echo "%$BACKUP_USER ALL =(hbase) NOPASSWD: /usr/bin/hbase" >> $BACKUP_USER
+sudo echo "## Allow $BACKUP_USER id to run commands needed for backup and restore" > $BACKUP_USER
+sudo echo "%$BACKUP_USER ALL =(hbase) NOPASSWD: /usr/bin/hbase" >> $BACKUP_USER
 
```

Comment:
sudo does no good here.  The file re-direct happens as the current user (output of sudo).